### ISSUE TITLE
www.icloud.com won't load with uMatrix enabled

### ISSUE THREAD

**Issue by rhavenn** [uBlockOrigin/uMatrix-issues#130]
### Prerequisites

- [X ] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
    - [X ] I also searched the existing issues at <https://github.com/gorhill/uMatrix/issues>
- [X ] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uMatrix](https://old.reddit.com/r/uMatrix/)
- I tried to reproduce the issue when...
    - [X ] uMatrix is the only extension
    - [ ] uMatrix with default lists/settings
    - [ ] using a new, unmodified browser profile
- [X ] I am running the latest version of uMatrix
- [X ] I checked the [documentation](https://github.com/gorhill/uMatrix/wiki) to understand that the issue I report is not a normal behavior
- [X ] I used the logger to rule out that the issue is caused by my ruleset

### Description

www.icloud.com won't load with uMatrix enabled in Firefox 66.0.2

### A specific URL where the issue occurs

https://www.icloud.com/

### Steps to Reproduce

Load www.icloud.com and whitelist everything or even disable uMatrix for scope on www.icloud.com and it never loads. Going into Firefox "AddOns" and marking uMatrix as disabled and the login form loads immediately. Re-enabling uMatrix and it stops working again.

### Ruleset

Stock ruleset. www.icloud.com disabled in scope. Temp rules attached.

### Supporting evidence

www.icloud.com throws an error after a minute or so. Screenshot attached.

### Your environment

* uMatrix version: 1.3.16
* Browser Name and version: Firefox 66.0.2
* Operating System and version: Windows 10, FreeBSD 12, Arch Linux

**Comment by rhavenn** [comment]
[umatrix_list.txt](https://github.com/uBlockOrigin/uMatrix-issues/files/3069461/umatrix_list.txt)

![www icloud com_connectionerror](https://user-images.githubusercontent.com/7809761/55970851-d433ed00-5c2c-11e9-88c8-3fcf02d66e1d.png)

**Comment by gorhill** [comment]
> I checked the documentation to understand that the issue I report is not a normal behavior

[About "the page is still broken after I created all necessary rules"](https://github.com/gorhill/uMatrix/wiki/About-%22the-page-is-still-broken-after-I-created-all-necessary-rules%22):

> Did you try to bypass the browser cache when you forced a reload of the page?

**Comment by rhavenn** [comment]
Dogh. I didn't even know those switches were a thing. My bad. It seems "Spoof Referrer" was the culprit. Sorry for the noise.

**Comment by GWKeys** [comment]
Let me start by saying Thank YOU! to Raymond HIll for such a wonderful add-on.  "I never leave home without it."  

This issue has been plaguing me for over a year now and I've finally taken some time today to either to report the bug or figure out what is going on.  I came here to report the bug and found rhaveen did all the hard work for me already bu submitting this ticket.  After reading gorhill's comment about switches, which I two didn't even realized existed or mattered, I turned off "Spoof Referrer", which immediately brought up icloud.com.  I'm now head-over heals for uMatrix once again.

I'm attaching a screen shot so others can easily find the location of those switches.  All the best and thanks for the help guys!

![Screenshot 2019-06-09 08 58 01](https://user-images.githubusercontent.com/27982758/59161233-fa430580-8a94-11e9-823e-3c96c4c1aeeb.png)
